client/overview/Overview.jsx

```diff
@@ -138,8 +138,8 @@ class Overview extends React.Component {
       return (
         <section className="o-product-overview">
           <ImageGallery selectedStyleId={this.state.selectedStyleId} stylePhotos={this.state.selectedStyle.photos} />
-          <ProductControls product={this.state.product} style={this.state.selectedStyle} />
-          <ProductDescription />
+          <ProductControls product={this.state.product} style={this.state.selectedStyle} styles={this.state.productStyles} />
```

Comment:
Consider using different name for the props named style and styles. Only because React uses a style attribute to designate inline styling. Not sure if it matters, but it could be confusing.
https://reactjs.org/docs/dom-elements.html#style